src/router/routes.ts

```diff
@@ -160,6 +160,18 @@ export const routes = [
       filingType: FilingTypes.REGISTRATION
     }
   },
+  {
+    path: '/dissolution-entity',
+    name: RouteNames.DISSOLUTION_ENTITY,
+    component: Views.DissolutionEntity,
+    meta: {
+      step: 1,
+      requiresAuth: true,
+      filingType: FilingTypes.VOLUNTARY_DISSOLUTION,
+      noStepper: true // to hide stepper for this view
```

Comment:
Will this default to false if a user find themselves on another route? 

To this point, we've always protected the routes between Dissolutions and Incorporations, not allowing temp reg numbers to access these routes, and vice versa. 

Might be worth looking into that mechanism in order to prevent Firms or Corporations from accessing each others routes, due to the fact they might?* share the same filing type (which i think we've used to differentiate in the past).

Reply:
It will be undefined (which is falsy) if it's not specified in the meta for another route. I'm OK with that.

I agree on the protection but it's actually pretty loose... in App.vue the code takes one path for Dissolutions and another page for IAs/Registrations. I don't have a better way in mind atm.

Reply:
Sure, I will check that 580, Currently, it's working when the filing type is 'SP'. Once we have the actual value I will refactor it, if needed.
Also from app.vue data will show according to filing type.

Reply:
Just make sure what you do works for SP dissolutions and doesn't interfere with SP registrations.

Faiz, you can resolve this conversation once you have read this comment 😀